web/src/lib/static/loadHousingMunicipalities.ts

```diff
@@ -0,0 +1,15 @@
+import { HousingMunicipality } from "@businessnjgovnavigator/shared/housing";
+import fs from "fs";
+import path from "path";
+
+const recordsDir = path.join(process.cwd(), "../shared/src/static");
+
+export const loadAllHousingMunicipalities = (): HousingMunicipality[] => {
+  const fullPath = path.join(recordsDir, "housing-municipalities.json");
+
+  const records = JSON.parse(fs.readFileSync(fullPath, "utf8")) as HousingMunicipalityRecords;
```

Comment:
Another great callout! Separated those into separate steps and added errors for when they can't be found, as well as an empty response object so the dependent components will at least get the empty in case of catastrophe (though we'll probably have bigger fish to fry at that point)